frontend/app-development/hooks/mutations/useUpdateProcessDataTypeMutation.ts

```diff
@@ -0,0 +1,24 @@
+import { type MetaDataForm } from '@altinn/process-editor/src/contexts/BpmnConfigPanelContext';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { updateProcessDataType } from 'app-shared/api/mutations';
+import { QueryKey } from 'app-shared/types/QueryKey';
+
+type useUpdateProcessDataTypeMutationPayload = {
+  form: MetaDataForm;
+};
+
+export const useUpdateProcessDataTypeMutation = (org: string, app: string) => {
+  const queryClient = useQueryClient();
+  debugger;
+  return useMutation({
+    mutationFn: ({ form }: useUpdateProcessDataTypeMutationPayload) => {
+      debugger;
+      console.log('form: ', form);
```

Comment:
Are line 15 and 16 supposed to be there?

Reply:
Removed 😅